ios/native/NativeProxy.mm

```diff
@@ -82,9 +82,11 @@ static id convertJSIValueToObjCObject(jsi::Runtime &runtime, const jsi::Value &v
   RCTBridge *bridge = _bridge_reanimated;
   REAModule *reanimatedModule = [bridge moduleForClass:[REAModule class]];
 
-  auto propUpdater = [reanimatedModule](jsi::Runtime &rt, int viewTag, const jsi::Object &props) -> void {
+  auto propUpdater = [reanimatedModule](jsi::Runtime &rt, int viewTag, const jsi::Value& viewName, const jsi::Object &props) -> void {
+    NSString *nsViewName = [NSString stringWithCString:viewName.asString(rt).utf8(rt).c_str() encoding:[NSString defaultCStringEncoding]];
+
     NSDictionary *propsDict = convertJSIObjectToNSDictionary(rt, props);
-    [reanimatedModule.nodesManager updateProps:propsDict ofViewWithTag:[NSNumber numberWithInt:viewTag] viewName:@"RCTView"];
+    [reanimatedModule.nodesManager updateProps:propsDict ofViewWithTag:[NSNumber numberWithInt:viewTag] viewName:nsViewName];
```

Comment:
`viewName` part does not match objc method naming convension which is that it when you read method with arguments then it creates a sort of a sentence, before we had:

update-props-of-view-with-tag-x

after your change:

update-props-of-view-with-tag-x-view-name

so clearly the last part does not fit, I'd suggest someting like:

update-props-of-view-with-tag-x-with-name-y

Reply:
fixed 6609d065d654dd05e644bee42dfdb62405907c7e